src/actions/posts.js

```diff
@@ -122,6 +122,14 @@ export function createPost(post, files = []) {
     };
 }
 
+export function resetCreatePost() {
```

Comment:
also can we rename the action to be `resetCreatePostRequest`

in this type of action you don't really need a thunk so you can do:

```js
export function resetCreatePostRequest() {
    return {type: PostTypes.CREATE_POST_RESET_REQUEST};
}
```